tests/terraform/runner/test_runner.py

```diff
@@ -449,10 +449,35 @@ def test_entire_resources_folder(self):
         valid_dir_path = current_dir + "/resources/"
         runner = Runner(db_connector=self.db_connector())
         if isinstance(runner.db_connector, IgraphConnector):
-            result = runner.run(root_folder=valid_dir_path, external_checks_dir=None, runner_filter=RunnerFilter())
-            self.assertEqual(len(result.passed_checks), 701)
-            self.assertEqual(len(result.failed_checks), 896)
-            self.assertEqual(len(result.skipped_checks), 7)
+            result = runner.run(root_folder=valid_dir_path, external_checks_dir=None, runner_filter=RunnerFilter(
+                checks=['CKV_AWS_21', 'CKV_AWS_42', 'CKV_AWS_62', 'CKV_AWS_53', 'CKV_AWS_18', 'CKV_AWS_61', 'CKV_AWS_144',
```

Comment:
why did you change it to include only certain checks? and less overall results?

Reply:
otherwise this test needs to be adjusted each time we add new checks